packages/synapse-interface/styles/global.css

```diff
@@ -195,3 +195,12 @@ body {
 .no-mt {
   margin-top: 0 !important;
 }
+
+.radial-gradient-underline {
```

Comment:
fy we can do this with tailwind + inline for consistency - example from landing page:

```
 <Button
  className={`
    border-[#AC8FFF] border text-sm
    px-4 py-3 hover:opacity-75
  `}
  style={{
    background: 'linear-gradient(310.65deg, rgba(255, 0, 255, 0.2) -17.9%, rgba(172, 143, 255, 0.2) 86.48%)',
    borderRadius: '10px',
  }}>
    Go to Explorer
</Button>
```

Reply:
great point - thanks for pointing that out! I'm leaning towards keeping it as a custom global class (need to read into tailwind's docs to see how I can set this natively within the tailwind system) just so it's easier to grab this specific styling for multiple buttons. I'm thinking these styles/colors will likely be applied across multiple components, so wanted to prevent having to find all the instances before updating

Reply:
For the app, we don't want to clutter up the global css file bc it will start to get tough to manage. Would prefer to use it if we have something super custom (like the loader animations, which we can't do in tw). There's some stuff in there for the landing page as well rn, which we should strip out too (all can be done in tw, and aren't repeated elsewhere either). We're also, atm, not re-using this css elsewhere and if it comes to the point we are, then we'll have a `<SpecialButton />` component instead.

Solution is just inline:

```
//removed radial-gradient-bg up here and added as inline style

const ConnectedButton = () => {
  const buttonClassName = `
  h-9 flex items-center justify-center w-36
  text-base text-white px-4 py-2 rounded-3xl
  text-center transform-gpu transition-all duration-75
  border-2 border-[#D747FF]
  hover:cursor-default
  `

  return (
    <button
      data-test-id="connected-button"
      className={buttonClassName}
      style={{
        background:
          'radial-gradient(249.66% 351.84% at -119.61% 97.91%, #FF00FF 0%, #AC8FFF 100%), linear-gradient(90deg, rgba(128, 0, 255, 0.2) 0%, rgba(255, 0, 191, 0.2) 100%)',
      }}
    >
      Connected
    </button>
  )
}
```

for `radial-gradient-underline` in `PortfolioTabManager`

```
style={{
  borderImage: 'linear-gradient(to right, rgba(255, 0, 255, 1), rgba(172, 143, 255, 1)) 1'
}}
```

Reply:
Updated in https://github.com/synapsecns/sanguine/pull/1104/commits/2cdfc16cba1cd60ddce9be3bb51d6f1a2abb95b0